libs/langchain/langchain/utilities/graphql.py

```diff
@@ -1,7 +1,8 @@
 import json
 from typing import Any, Callable, Dict, Optional
-
 from pydantic import BaseModel, Extra, root_validator
+from gql import Client
```

Comment:
these are optional dependencies, so they should be tucked into optional imports , a user without `gql` installed should be able to import this file without getting any side effects